source/options.tsx

```diff
@@ -89,7 +89,7 @@ async function init(): Promise<void> {
 		}
 	}
 
-	browser.storage.local.set({featuresAlreadySeen});
+	await browser.storage.local.set({featuresAlreadySeen});
```

Comment:
@fregante I assume this is correct?

Reply:
No since this delays unrelated features.
```suggestion
	void browser.storage.local.set({featuresAlreadySeen});
```

I actually split this `init` function into multiple parts before, but then I undid it because it was unrelated to the PR. I'll open a new PR for that later